tests/test_cli_compile.py

```diff
@@ -107,6 +107,27 @@ def test_command_line_setuptools_output_file(
     assert os.path.exists(expected_output_file)
 
 
+def test_command_line_setuptools_nested_output_file(pip_conf, tmpdir, runner):
+    """
+    Test the output file for setup.py in nested folder as a requirement file.
+    """
+    proj_dir = tmpdir.mkdir("proj")
+
+    with open(os.path.join(str(proj_dir), "setup.py"), "w") as package:
```

Comment:
Oh, I've missed that one:

```suggestion
    with open(str(proj_dir / "setup.py"), "w") as package:
```